implementations/c/external/cryptoauth/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+if (NOT OCKAM_ENABLE_ATECC608A_BUILD)
```

Comment:
@SanjoDeundiak what was the reason to move cryptoauthlib to cryptoauth?
I wonder if this breaks the ability to find and externally installed cryptoauthlib

Reply:
@mrinalwadhwa even though header is called cryptoauthlib.h, the original target name from Microchips cmake files is cryptoauth. Honestly, I'm not 100% sure it's the right thing to do, we still have some occasional problems with this dependency which should be investigated in the future, so eventually we're gonna come back to this question.

Reply:
cool, let's log an issue so we remember to test that externally installed libs should continue to work.

Reply:
@mrinalwadhwa we have them
https://github.com/ockam-network/ockam/issues/351
https://github.com/ockam-network/ockam/issues/291